pennylane/measure.py

```diff
@@ -114,9 +114,9 @@ def circuit(x):
         # delete operations from QNode queue
         if isinstance(op, Tensor):
             for o in op.obs:
-                qml._current_context.queue.remove(o)
+                _remove_if_in_queue(o)
```

Comment:
What was the bug that was fixed here? :slightly_smiling_face:

Reply:
At one point, `expval` was updated to remove an observable only if it was already in the queue. However, `var` and `sample` had not yet been updated, so attempting to add `var(obs)` if `obs` didn't happen to already be in the queue would cause an exception.